tools/updateDerivativesTest.php

```diff
@@ -1,171 +0,0 @@
-#!/usr/bin/php
```

Comment:
@driusan deleting this as it is deemed unnecessary by the PR that is creating it (read conversation in https://github.com/aces/Loris/pull/7433)

@AlexandraLivadas please confirm

Reply:
@ridz1208 Yes this is unnecessary and can be deleted!